src/data/store/slices/linkSaturation.slice.ts

```diff
@@ -32,12 +31,19 @@ const linkSaturationSlice = createSlice({
         updateLinkSaturation: (state, action: PayloadAction<number>) => {
             state.linkSaturationTreshold = action.payload;
         },
-        updateTotalOPs: (state, action: PayloadAction<{ temporalEpoch: number; totalOps: number }>) => {
-            const { temporalEpoch, totalOps } = action.payload;
+        updateTotalOPs: (
```

Comment:
same as for the selector. please separate into two separate reducers. there is no good reason why they are the same method